### ISSUE TITLE
Proof of concept journal IPOL

### ISSUE THREAD

**Issue by jhergon** [mlcommons/ck#617]
The source code of ML/AI algorithms for image processing is published in the open journal IPOL. The idea is to be able to convert this type of published code to the CK Framework, so that it can be interoperable, automatable, portable, reproducible and easily incorporated into MLops workflows. This As a proof of concept and the generation of a guide tutorial adapted for this type of publication for future autors

for example:
http://www.ipol.im/pub/art/2022/439/
which uses a deep learning algorithm

**Comment by gfursin** [comment]
Thank you very much, Jose, for opening this ticket and Happy New Year!
We are a bit swamped working with the community on MLPerf inference v3.0 submissions but I have added to our ToDo list and want to have a look at it as soon as we stabilize everything for the submission! 
Will keep in touch!

**Comment by gfursin** [comment]
Hi Jose, 
It was great to meet you in person at ENS yesterday and thank you for a cool hackathon. 
It was cool to see that we can quite quickly reproduce a journal article and provide a unified CK2 (CM) interface to it to run on different machines. I need to clean it up a bit and I will share it with you for further discussions ...
Will keep in touch!

**Comment by gfursin** [comment]
Hi Jose,
We are finishing the release of the MLPerf inference v3.0 results (next week) and will resume our community developments.
I will commit our CK2/CM workflow for the IPOL paper then and will be happy to resume our discussions.
In the meantime, we have just released an open-source platform for optimization and reproducibility challenges that we will be developing as a community effort too. I have added our "challenge" there: https://access.cknowledge.org/playground/?action=challenges&name=reproduce-and-automate-ipol-paper ... 
Hope everything is fine on your side and will keep in touch!
